src/webpackConfigExtension.ts

```diff
@@ -48,9 +48,18 @@ export default (
     },
     module: {
       rules: [
+        // Allow importing files as a raw string
+        // import foo from 'foo.ts?raw';
+        {
+          resourceQuery: /raw/,
```

Comment:
it always makes me uncomfortable when a regex doesn't have any anchors at beginning or end, since `/raw/` would probably also match the string `"foo.ts?crawfish"` 🦞

Here's an example from webpack tests where they use a `\?` at the beginning and `$` at the end: https://github.com/webpack/webpack/blob/7634cd28bf67a3e0e4608a5ae496ecb474b66d6a/test/configCases/css/local-ident-name/webpack.config.js#L20